CharacterCreation/03_Feats.txt

```diff
@@ -351,6 +351,23 @@ buzzed or hammered, you deal extra damage equal to your strength modifier. These
 bonuses only apply to melee attacks. You also get a +10% on rolls to avoid 
 blacking out.  
 
+== Point Blank ==
+
+Prerequisites:
+
+*  Backstab feat
+*  Profeciency with Pistols or SMGs
+
+    It's hard to miss if you shoot from less than a foot away. Luckily, you're
+sneaky enough that you end up that close with some frequency. You tend to target
+the gaps in armor around the neck or eyes, sometimes adding a quick punch or 
+fast kick to help contort your target into a vulnerable stance. 
+
+    You can use backstab with suppressed CQB guns as long as you're in melee
```

Comment:
I would suggest instead of 'suppressed CQB guns' `CQB guns with a suppressor attached`, to help out our hypothetical players who may be less familiar with firearms terminology.